browser.jsx

```diff
@@ -1,9 +1,9 @@
 'use strict'
-var remote = require('remote')
-var Menu = remote.require('menu')
-var MenuItem = remote.require('menu-item')
-var clipboard = require('clipboard')
-var urllib = require('url')
+var remote = require('electron').remote
+var Menu = require('electron')
```

Comment:
Sorry, i'm not sure what your question is. 

But without remote the minimize, maximize controls don't work.